xmodule/modulestore/tests/test_mixed_modulestore.py

```diff
@@ -109,6 +111,10 @@ class CommonMixedModuleStoreSetup(CourseComparisonTest):
         ],
         'xblock_mixins': modulestore_options['xblock_mixins'],
     }
+    ENABLED_OPENEDX_EVENTS = [
+        "org.openedx.content_authoring.xblock.deleted.v1",
+        "org.openedx.content_authoring.xblock.published.v1",
+    ]
 
     def setUp(self):
```

Comment:
Same comment as above. You're missing the setup class

Reply:
Done.